internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/endToEnd/TestDeferredVersionSwap.java

```diff
@@ -496,4 +502,83 @@ public void testDeferredVersionSwapWithHybridStore() throws IOException {
     }
   }
 
+  @Test(timeOut = TEST_TIMEOUT)
+  public void testDeferredVersionSwapWithoutTargetRegionPush() throws IOException {
```

Comment:
what is the purpose of this test? should we test when there not enough replica the deferred version swap will wait for them to come back up?

Reply:
This is just a generic deferred version swap test which we don't have for a batch push store. This also validates that the future version is set to online after push job completion and just the version swap is done during roll forward. This can be further improved with checking whether the deferred swap fails once we add the check for min active replica. For now, this is a generic deferred swap test.

Reply:
moved it to a new file and renamed the existing file to `TestDeferredVersionSwapWithTargetedRegionPush.java`  to keep the target region push tests and this separate.